### PR TITLE
Ocrvs 1619 Forgotten pin flow

### PR DESCRIPTION
**Pull request by tahmidrahman-dsi**
forgotPINFlow

### CODE REVIEW

**Comment by tahmidrahman-dsi** [comment]
**Test reports**

Passes unit tests locally

image

File: packages/client/src/views/Unlock/ForgotPIN.tsx [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+                  {intl.formatMessage(buttonMessages.forgotPassword)}
+                </ForgotPasswordLink>
+              </ActionWrapper>
+            </StyledForm>
+          </>
+        )}
+      </Container>
+    </PageWrapper>
+  )
+}
```
Comment (euanmillar) [inline packages/client/src/views/Unlock/ForgotPIN.tsx:272]: Love the React hook!

**Review by euanmillar — APPROVED** [review]
Great work Tahmid!

**Comment by euanmillar** [comment]
LGTM!

**Pull request merged**